ncolony/ctllib.py

```diff
@@ -39,23 +39,81 @@
 import json
 import os
 
+import attr
 from twisted.python import filepath
 
 from ncolony import main as mainlib
 
 NEXT = functools.partial(next, itertools.count(0))
 
-Places = collections.namedtuple('Places', 'config messages')
-
+@attr.s(frozen=True)
+class Places(object):
+    config = attr.ib()
+    messages = attr.ib()
 
 def _dumps(stuff):
     return json.dumps(stuff).encode('utf-8')
 
+_COMMAND_LINE = object()
```

Comment:
i would document this, seems weird to me that a random empty `object` is used to key a metadata dict